advanced/Scripts/webpage.sh

```diff
@@ -636,8 +636,11 @@ Interfaces:
 
 Teleporter() {
     local datetimestamp
+    local host
     datetimestamp=$(date "+%Y-%m-%d_%H-%M-%S")
-    php /var/www/html/admin/scripts/pi-hole/php/teleporter.php > "pi-hole-teleporter_${datetimestamp}.tar.gz"
+    host=$(hostname)
+    host="${host//./_}"
+    php /var/www/html/admin/scripts/pi-hole/php/teleporter.php > "pi-hole-${host}-teleporter_${datetimestamp}.tar.gz"
```

Comment:
presumably if `${host}` is blank here (I can't see why it would be), we would end up with a `pi-hole--teleporter...` file?

Reply:
I think so. Have not tried it myself, but that's what I expect to see. Does not look beautiful but would not be a good reason to me to introduce a function to prevent this edge case.

Reply:
```suggestion
    hostname=$(hostname)
    host="${hostname//./_}"
    php /var/www/html/admin/scripts/pi-hole/php/teleporter.php > "pi-hole-${host:-noname}-teleporter_${datetimestamp}.tar.gz"
```

Reply:
I do like a cheeky bash-ism when I see one. Does that say "if `host` is empty then state `noname`"?

Reply:
Yeah, and really it could be done without changing the variable names:

```suggestion
    host=$(hostname)
    host="${host//./_}"
    php /var/www/html/admin/scripts/pi-hole/php/teleporter.php > "pi-hole-${host:-noname}-teleporter_${datetimestamp}.tar.gz"
```

Reply:
Or even: ?

```suggestion
    host="${hostname//./_}"
    php /var/www/html/admin/scripts/pi-hole/php/teleporter.php > "pi-hole-${host:-noname}-teleporter_${datetimestamp}.tar.gz"
```

Reply:
That doesn't actually call `hostname` though. You still need the first line to execute the `hostname` call.

Reply:
The `:-noname` is really nice. Didn't know that you can put in default values.